pyart/aux_io/kazr_spectra.py

```diff
@@ -67,129 +75,162 @@ def read_kazr_spectra(filename, field_names=None, additional_metadata=None,
     """
     if not _XARRAY_AVAILABLE:
         raise MissingOptionalDependency(
-            "Xarray is required to read KAZR spectra files "
-            "but is not installed!")
+            "Xarray is required to read KAZR spectra files " "but is not installed!"
```

Comment:
The two quoted strings next to eachother can be combined.

Reply:
Good catch